content/pages/healthcare/apply/application.md

```diff
@@ -3,9 +3,10 @@ title: Apply for Health Care
 entryname: hca
 layout: page-react.html
 body_class: page-healthcare
-in_maintenance: false
-maintenance_line1: We're sorry. The healthcare application is currently down while we fix a few things. We will be back up as soon as we can.
-maintenance_line2: In the meantime, you can still call 1-877-222-VETS(8387) and press 2 to complete this application over the phone.
+in_maintenance: true
+maintenance_line1: Scheduled Maintenance
```

Comment:
We don't want to keep the original language and just replace "down while we fix a few things" with "under scheduled maintenance"? So:
```
We're sorry. The health care application is currently under scheduled maintenance.
Please try again after 12pm EST Sunday, January 8th.
```
Or something like that?

Also wondering if we want to commit to an exact time. Just in case maintenance runs long, is there another way to say when they should come back without naming a time? Like "Please try again in a few hours"? Or "Please try again in the afternoon on Sunday, January 8th"?